pkgs/applications/misc/kjv/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "kjv";
+  version = "20190311";
+
+  src = fetchFromGitHub {
+    owner = "LukeSmithxyz";
```

Comment:
why package a fork instead of the main repository? https://github.com/bontibon/kjv

Reply:
it contains another book in the bible

Reply:
why they didn't do a PR to merge it back in, idk

Reply:
I think the end-user could [override it](https://nixos.org/nixos/nix-pills/nixpkgs-overriding-packages.html) with the [patch](https://github.com/LukeSmithxyz/kjv/commit/b92b7622285d10464f9274f11e740bef90705bbc)
```nix
  kjv = pkgs.kjv.overrideAttrs (attrs: {
    patches = attrs.patches ++ [ /path/to/apocrypha.patch ];
  });
```

Reply:
It seems fine and simpler to me to package the fork, and would prefer to stick with that.  Just let me know if you don't consider that to be acceptable.

Reply:
How about just fetching the patches from the fork and applying them?